docs/SVGTemplateRender.md

```
@@ -0,0 +1,158 @@
+|-----------------------|-----------------------------------------------------------------------------------------------------------|
+| renderMethod          | Parent object describing how to render the VC                                                             |
+| type                  | Method type (e.g., TemplateRenderMethod)                                                                  |
+| renderSuite           | Rendering suite, e.g., svg-mustache                                                                       |
+| template              | Inline or external template definition                                                                    |
+| id                    | URL pointing to the SVG template stored by the issuer                                                     |
+| mediaType             | MIME type of the template (e.g., image/svg+xml)                                                           |
+
+
+### Common Media Types Supported
+
+| MediaType	            | Usage                                                                                                     | 
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 12

(MD010, no-hard-tabs)